README.rst

```diff
@@ -768,6 +768,33 @@ By default ``sops`` just dumps all the output to the standard output. We can use
 Beware using both ``--in-place`` and ``--output`` flags will result in an error.
 
 
+Using the publish command
+~~~~~~~~~~~~~~~~~~~~~~~~~
+``sops publish $file`` publishes a file to a pre-configured destination (this lives in the sops
+config file). Additionally, support re-encryption rules that work just like the creation rules.
+
+This command requires a ``.sops.yaml`` configuration file. Below is an example:
+
+.. code:: yaml
+
+   creation_rules:
+      - pgp: 1022470DE3F0BC54BC6AB62DE05550BC07FB1A0A
+   destination_rules:
+      - s3_bucket: "sops-secrets"
+        path_regex: s3/*
+        reencryption_rule:
+           pgp: F69E4901EDBAD2D1753F8C67A64535C4163FB307
```

Comment:
I wonder if it would be better to name it `creation_rule` while still having it nested like it is? This would also require handling all of the features that creation rules currently support. Potentially, if this is done along side something like https://github.com/mozilla/sops/pull/473#discussion_r291751560, it could assist users in better understanding what changes will be made when they use `sops publish`.

But I'm not sure, need to think about this one a little bit...

Reply:
if you're going to reuse the `creation_rule`, perhaps calling it `recreation_rule` would be clear enough?